recipes/splign/meta.yaml

```diff
@@ -0,0 +1,37 @@
+{% set version = "2.1.0" %} # Remember to update sha256 below
+
+package:
+  name: splign
+  version: {{ version }}
+
+build:
+  noarch: generic
+  number: 0
+  script: mkdir -p $PREFIX/bin ; mv * $PREFIX/bin
+  
+  
+
+source:
+  url: https://ftp.ncbi.nlm.nih.gov/genomes/TOOLS/splign/linux-i64/splign.tar.gz
+  sha256: f3912bf15e42cbf27988dad28be5d6ba2551daa5cd4fa4b88cfe4aa039ca82db
+
+requirements:
+  run:
+    - bzip2
+    - lmdb
+    - libuv
+    - pcre
+    - libnghttp2
+    - sqlite
+    - bz2file
```

Comment:
Please remove this dependency: `bz2file` is a Python package that wraps bzip2. The correct dependency is just `bzip2`.